### PR TITLE
Update theme

### PR DESCRIPTION
**Pull request by itsacoyote**
# What :computer: 
* Update theme with correct usage of colors

# Why :hand:
* Match current branding

# Evidence :camera:
Screenshot 2024-05-14 at 12 09 27 PM

### CODE REVIEW

File: components/HeaderComponent.vue [hunk 1]
```diff
@@ -8,11 +8,55 @@ defineProps<{
...
+      ]
+    : [
+        {
+          label: 'Kitchen Sink',
+          to: '/kitchen-sink',
+        },
+        {
+          label: 'Page',
+          to: '/about-us',
+        },
```
Comment (dutterbutter) [inline components/HeaderComponent.vue:50]: ?

File: nuxt.config.ts [hunk 2]
```diff
@@ -58,4 +58,28 @@ export default defineNuxtConfig({
...
     },
   },
+  $development: {
+    runtimeConfig: {
+      public: {
+        app: 'template',
+        urls: {
+          docs: 'https://staging-docs.zksync.io',
+          code: 'https://staging-code.zksync.io',
+          sdk: 'https://staging-docs.zksync.io/sdk',
```
Comment (dutterbutter) [inline nuxt.config.ts:68]: Might also need to include: `https://zksync-sdk-docs-staging.web.app/sdk` until the domain is fixed.

File: components/HeaderComponent.vue (hunk 1 above)
Comment (itsacoyote) [reply components/HeaderComponent.vue:50]: These are test pages for this template, they don't show up in the other projects when they use this. This allows us to develop and design in development in this repo before publishing.

File: nuxt.config.ts (hunk 2 above)
Comment (itsacoyote) [reply nuxt.config.ts:68]: Ah, good idea, I'll add that in for the staging.

**Review by dutterbutter — DISMISSED** [review]

**Review by dutterbutter — APPROVED** [review]

**Pull request merged**